cns/restserver/restserver.go

```diff
@@ -57,8 +59,8 @@ type allocatedIPCount struct {
 // containerstatus is used to save status of an existing container
 type containerstatus struct {
 	ID                            string
-	VMVersion                     string
-	HostVersion                   string
+	DncNCVersion                  string
```

Comment:
Wont this break backcompat? CNS is already running in production and JSON file stored has previous names, how will you unmarshel it back to this new struct? Also I dont understand what does DncNCVersion means?

Reply:
Also not sure if this VMCersion is NC version. I believe it is the VM Version only. We are not populating it for AKS scenario. Actual NC version is inside the CreateNetworkContainerRequest

Reply:
1. It's backward compatible. No unmarshal happens in containerstatus.
2. DNCNCVersion means NC version which coming from DNC. This PR is resolving the issue when discrepancy happens in NC version from DNC and from NMAgent.
3. VMVersion is NC version. It get value from nc request version and compared with nc request version. Code reference 
https://github.com/Azure/azure-container-networking/blob/master/cns/restserver/util.go#L135
https://github.com/Azure/azure-container-networking/blob/master/cns/restserver/api.go#L789

Reply:
After discussion, we'll remain the VMVersion variable name here.

Reply:
Remained it as VMVersion here.